recipes/srreal/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "srreal" %}
+{% set version = "v1.2" %}
```

Comment:
Replace with "1.2", the `v` is in tag name, but package version should not include it.